docs/antctl.md

```diff
@@ -98,6 +99,18 @@ integer):
 antctl log-level LEVEL
 ```
 
+### Showing feature gates status
+
+Antrea supports showing feature gates of Antrea Controller or Agent using the `antctl get featuregates`
```

Comment:
How about: The feature gates of Antrea Controller and Agent can be shown using the `antctl get featuregates` command?